scilifelab_epps/wrapper.py

```diff
@@ -80,7 +96,7 @@ def epp_wrapper(args):
 
             # On script success
             else:
-                logging.info("Script completed successfully.")
+                logging.info("Script finished successfully. Uploading log file.")
                 logging.shutdown()
                 upload_file(
                     file_path=log_filename,
```

Comment:
`sys.stderr.write(
"Script finished successfully, but log contains errors or warnings, please have a look."`

Wouldn't all the error messages already be displayed in the header? Do we need to read the log file again?

Reply:
It's possible to have multiple warnings and errors for different samples, so it's easier to reference the file than to try to fit all information into the header.

Reply:
Ah sorry, I meant do we need to read the whole log again here
`log_content = open(log_filename).read()`

Reply:
We could use a custom handler, needs testing though
```
class ErrorWarningHandler(logging.Handler)
    
    errors_or_warnings = False

    def __init__(self):
        super().__init__()

    def emit(self, record):
        # This method is called for each log message        
        # Check if the message contains "ERROR:" or "WARNING:"
        if record.levelname=="ERROR:" or record.levelname=="WARNING:":
            # If there's an error or warning, set the flag to True
            errors_or_warnings = True

Reply:
Implemented in [59e2d62](https://github.com/NationalGenomicsInfrastructure/scilifelab_epps/pull/406/commits/59e2d62f3b058b5872056530ac4a781c39c99edf) and [1005c11](https://github.com/NationalGenomicsInfrastructure/scilifelab_epps/pull/406/commits/1005c119eb24b161658bd6ba40fe6aab6d204fd5).

See here for test steps
https://ngi-lims-stage.scilifelab.se/clarity/work-details/1062599
https://ngi-lims-stage.scilifelab.se/clarity/work-details/1062133